articles/install-guide/index.md

```diff
@@ -1,53 +1,225 @@
 ---
-title: Getting Started with the Microsoft Quantum Development Kit
-author: anpaz-msft
-ms.author: andres.paz@microsoft.com 
-ms.date: 3/26/2019
+title: Learn how to install the Microsoft Quantum Development Kit (QDK)
+author: natke
+ms.author: nakersha
+ms.date: 9/23/2019
 ms.topic: article
+ms.custom: how-to
 uid: microsoft.quantum.install
 ---
 
-# Getting Started with the Microsoft Quantum Development Kit #
+# Install the Microsoft Quantum Development Kit (QDK)
 
-## Choose the Environment That's Right for You ##
+Learn how to install the Microsoft Quantum Development Kit (QDK), so that you can get started with quantum programming. The QDK consists of:
 
-### Jupyter Notebooks ###
+- the Q# programming language
+- a set of libraries that abstract complex functionality in Q#
+- a host application (written in Python or a .NET language) that runs quantum operations written in Q#
+- tools to facilitate your development
 
- A great way to get started with Q# before installing the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git).  In particular:
+Depending on your chosen development environment, there are different installation steps. Choose your environment from the sections below.
 
-* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/Microsoft/Quantum/master?filepath=Samples%2Fsrc%2FIntroToIQSharp%2FNotebook.ipynb): explains how to compile and simulate Q# operations inside a Jupyter notebook.
-* **[Teleport](https://github.com/Microsoft/Quantum/tree/master/Samples/src/Teleportation/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/microsoft/Quantum/master?filepath=Samples%2Fsrc%2FTeleportation%2FNotebook.ipynb): shows how to implement the teleport algorithm using Q#.
+## Develop with Python
 
-To run these locally, take a look at our [Jupyter notebooks getting started guide](xref:microsoft.quantum.install.jupyter).
+1. Pre-requisites
 
-### Visual Studio ###
+    - [Python](https://www.python.org/downloads/) 3.6 or later
+    - The [anaconda](https://docs.anaconda.com/anaconda/install/) package manager
 
- If you are already a [Visual Studio](https://visualstudio.microsoft.com/vs/) user, you can install the extension to get started developing Q# in Visual Studio. 
-> [!div class="button"]
-> [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+1. Install the `qsharp` package
 
-Then, check our [quickstart guide](xref:microsoft.quantum.write-program?tabs=tabid-vs2019) for step-by-step instructions for your first quantum program.
+    ```bash
+    conda install -c quantum-engineering qsharp
+    ```
 
-### Visual Studio Code ###
+1. Install the `iqsharp` package
```

Comment:
This command will work, but isn't strictly needed, given that `qsharp` has a dependency on `iqsharp`.

Reply:
Fixed